.golangci.yaml

```diff
@@ -24,30 +24,19 @@ linters-settings:
     enabled-checks:
```

Comment:
What's the reason for removing these? Just invalid/old?

Reply:
Yeah they're covered by other linters, a bunch of warnings about already enabled so I removed them